medical/models/medical_appointment.py

```diff
@@ -97,39 +101,44 @@ class MedicalAppointment(orm.Model):
     def create(self, cr, uid, vals, context=None):
         val_history = {}
         ait_obj = self.pool.get('medical.appointment.history')
-        date_time_str = vals['appointment_day'] + ' ' + vals['appointment_hour'] + ':' + vals['appointment_minute']
-        vals['appointment_date'] = datetime.strptime(date_time_str, '%Y-%m-%d %H:%M')
+        date_time_str = vals['appointment_day'] + ' ' + \
+            vals['appointment_hour'] + ':' + \
+            vals['appointment_minute']
+        vals['appointment_date'] = datetime.strptime(date_time_str,
+                                                     '%Y-%m-%d %H:%M')
 
         val_history['name'] = uid
         val_history['date'] = time.strftime('%Y-%m-%d %H:%M:%S')
-        val_history['action'] = "--------------------------------  Changed to Comfirm  ------------------------------------\n"
-
+        val_history['action'] = "--------------------------------"
```

Comment:
See my refactoring here 
https://github.com/OCA/vertical-medical/pull/6/files#diff-3342df827adae2a46375586cb86b0046R29

    HISTORY_ACTION_FORMAT = "-" * 32 + "  %s  " + "-" * 32 + "\n"

Then:

 val_history = {
            'name': uid,
            'date': time.strftime('%Y-%m-%d %H:%M:%S'),
            'action': HISTORY_ACTION_FORMAT % "Changed to Confirm"
        }